app/src/components/user_schema_table.rs

```diff
@@ -153,7 +154,27 @@ impl UserSchemaTable {
         let hardcoded = ctx.props().hardcoded;
         html! {
             <tr key={attribute.name.clone()}>
-                <td>{&attribute.name}</td>
+                <td>
+                    {&attribute.name}
+                    {
+                        match validate_attribute_name(&attribute.name) {
+                            Ok(()) => {
+                                html!{}
+                            }
+                            Err(_invalid_chars) => {
+                                html!{
+                                    <>
+                                        <br/>
+                                        <small class="text-warning">
+                                            {"Warning: This attribute uses one or more invalid characters. "}
```

Comment:
Can we refactor this code? If not a component, at least put the error message in a constant somewhere

Reply:
Done.